### PR TITLE
Mayhem Integration

### PR DESCRIPTION
**Pull request by code-terror**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: No need to copy the Mayhemfile inside of the container.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+project: code-terror/evm
```
Comment (xansec) [inline Mayhemfile:1]: Can you make the project name generic?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: Dockerfile (hunk 1 above)
Comment (code-terror) [reply Dockerfile:1]: Done!

File: Mayhemfile (hunk 2 above)
Comment (code-terror) [reply Mayhemfile]: Done

**Comment by code-terror** [comment]
@xansec ...pushed the changes.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**